source/Integrate/Tutorials/analytics_with_keen_io.md

```diff
@@ -92,10 +99,14 @@ Congratulations! You’ve created your first dashboard!
 
 Now start digging into your data and creating more custom analyses & visualizations.
 
+{% anchor h3 %}
+Doing Drilldowns and Reporting on Your Data
+{% endanchor h3 %}
 
-## <a name="-doing-drilldowns-and-reporting-on-your-data"></a>Doing Drilldowns and Reporting on Your Data
+{% anchor h4 %}
+Select_Unique
+{% endanchor h4 %}
 
-### Select_Unique
 `Select_Unique` allows you extract many properties from a single field and filter on a specific segment of users based on any number of fields such as campaigns, clicks, and companyID.
```

Comment:
Should Select_Unique be in quotes or a different font? I'm stuck on how we should differentiate our UI elements/field names when there's not an action.